source/02-FN/01.ptx

```diff
@@ -71,7 +71,7 @@
     <definition xml:id="def-function">
       <statement>
         <p>
-          A <term>function</term> is a relation where every input (or <m>x</m>-value) is mapped onto <term>**exactly one**</term> output (or <m>y</m>-value).</p>
+          A <term>function</term> is a relation where every input (or <m>x</m>-value) is mapped onto <em>exactly one</em> output (or <m>y</m>-value).</p>
```

Comment:
Only use `<term>` when introducing new terminology, and use `<em>` when you want to emphasize something. They look the same in HTML, but make a big difference to the (say) braille reader.